### ISSUE TITLE
Outdated luis_schema_version when parsing .lu file

### ISSUE THREAD

**Issue by ionut-matei** [microsoft/botbuilder-tools#1259]
## Tool
Name: Ludown
Version: 1.3.2
OS: Windows 10

## Describe the bug
The JSON output from _ludown parse toluis_ command contains an outdated value for the _luis_schema_version_ property.
## To Reproduce
1. Run the _ludown parse toluis_ command on this .lu file
```text
# Intent
- test
```
2. The generated JSON file contains an outdated value for the _luis_schema_version_ property. It should be 3.2.0, not 3.0.0
```json
{
  "intents": [
    {
      "name": "Intent"
    }
  ],
  "entities": [],
  "composites": [],
  "closedLists": [],
  "regex_entities": [],
  "model_features": [],
  "regex_features": [],
  "utterances": [
    {
      "text": "test",
      "intent": "Intent",
      "entities": []
    }
  ],
  "patterns": [],
  "patternAnyEntities": [],
  "prebuiltEntities": [],
  "luis_schema_version": "3.0.0",
  "versionId": "0.1",
  "name": "demo",
  "desc": "",
  "culture": "en-us"
}
```
3. If I export an application from the LUIS portal, the value for the _luis_schema_version_ property is 3.2.0.

## Expected behavior
The same behaviour as exporting the application from the LUIS portal.
```json
....
"luis_schema_version": "3.2.0" 
....
```

**Comment by sgellock** [comment]
@ionut-matei thank you for opening the issue.  we'll work to fix this in our 4.6 release.

**Comment by sgellock** [comment]
@munozemilio @vishwacsena when we fix this lets put the fix into the Bf-CLI extension
